build.gradle

```diff
@@ -16,11 +20,10 @@ version = '2.1.0'
 apply plugin: 'com.android.library'
 apply plugin: 'checkstyle'
 apply plugin: 'com.f2prateek.checkstyle'
-apply plugin: 'android-maven'
 
 android {
   compileSdkVersion 23
```

Comment:
does it matter if SDK version is different than our project?

Reply:
This worked before, so I don't think so. I'll bump it anyway.

Reply:
ok